### ISSUE TITLE
ISP child safe feature blocks obs.ninja

### ISSUE THREAD

**Issue by JBraddockm** [steveseguin/obsninja#155]
I wanted to try OBS.ninja with a friend - in the UK - and apparently Child Safe feature of his ISP - Virgin Media - blocks OBS.ninja. I was wondering if this is due to ninja domain?

This wouldn't be a problem for a host. I am sure that anyone who understands how to use OBS.ninja can easily remove this restriction but for remote guests, this could be a problem as Child Safe is on by default in many ISPs.

I'll ask other friends with different ISPs to try and replicate the issue.

![Virgin Media - Child Safety](https://user-images.githubusercontent.com/16445610/82450148-399b1400-9aa4-11ea-93b7-b0875757f053.jpeg)

**Comment by unknowndomain** [comment]
You'll have better results by clicking the "Don't think this page should be blocked? Let us know." link because there is nothing you can do about this as a host, but as a customer you can query this with your ISP.

**Comment by JBraddockm** [comment]
True. My worry is that if this is a common issue with other ISPs, this could potentially cause problems for guests. I'll see if I can replicate the issue with other ISPs in the UK. I can then try to talk to them individually.

**Comment by unknowndomain** [comment]
It’s far more likely that it’s blocked for children as it’s effectively video conferencing software.

**Comment by unknowndomain** [comment]
It might be helpful to find out what dataset Virgin Media uses for this, as I doubt they maintain their own block lists.

**Comment by steveseguin** [comment]
Using a VPN like Speedify might help solve this issue.

I haven't found a solution yet otherwise --

**Comment by steveseguin** [comment]
Closing old issue automatically by bot; the old obs.ninja domain issue no longer applies.
